dbms/src/IO/Progress.h

```diff
@@ -6,26 +6,44 @@
 
 #include <Core/Defines.h>
 
-
 namespace DB
 {
 
 class ReadBuffer;
 class WriteBuffer;
 
-
 /// See Progress.
 struct ProgressValues
 {
     size_t rows;
     size_t bytes;
     size_t total_rows;
+    size_t write_rows;
+    size_t write_bytes;
 
     void read(ReadBuffer & in, UInt64 server_revision);
     void write(WriteBuffer & out, UInt64 client_revision) const;
     void writeJSON(WriteBuffer & out) const;
 };
 
+struct ReadProgress
+{
+    size_t rows;
```

Comment:
Let's also rename here and above for consistency: read_rows, read_bytes, total_rows_to_read.